spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -806,6 +806,12 @@ deployer.<app>.kubernetes.livenessProbePath=/actuator/health
 deployer.<app>.kubernetes.readinessProbePath=/actuator/info
 ```
 
+To automatically set both _liveness_ and _readiness_ endpoints per application to the Spring Boot 2.x paths, the following property can be set:
```

Comment:
to the 'default' Spring Boot 2.x paths....

Reply:
updated